libs/sql-schema-describer/src/mysql.rs

```diff
@@ -437,9 +437,9 @@ impl SqlSchemaDescriber {
             SELECT DISTINCT
                 index_name AS index_name,
                 non_unique AS non_unique,
-                column_name AS column_name,
+                Binary column_name AS column_name,
```

Comment:
I don't know that binary qualifier, is it something like a coercion?

Reply:
That forces the comparisons to happen in a case sensitive way, without it Distinct collapses ADDRESSID, AddressId and addressId into one -.-

Reply:
ugh